otter/json_schema/group_schemas.py

```diff
@@ -87,6 +87,10 @@
     }
 }
 
+_bfv2_server = deepcopy(_bfv_server)
+bdm = _bfv2_server["properties"].pop("block_device_mapping")
```

Comment:
it seems to follow the convention this variable should also be prefixed with an `_`?